narwhals/expr.py

```diff
@@ -3741,6 +3742,53 @@ def func(plx: Any) -> Any:
     return Expr(func)
 
 
+def nth(*indices: int | Sequence[int]) -> Expr:
+    """
+    Creates an expression that references one or more columns by their index(es).
+
+    Warning:
+        `nth` is not supported for Polars version < 0.20.26. Please use `col` instead.
```

Comment:
You can refer to `col`, but I am afraid we need to add plugin in `mkdocs.yml` file.
For here
```suggestion
        `nth` is not supported for Polars version<0.20.26. Please use [`col`](/api-reference/narwhals/#narwhals.col) instead.
```

In `mkdocs.yml`

```bash
plugins:
- search
- autorefs
- mkdocstrings:
  ...
```

and in `/docs/requirements-doc.txt`

```diff
mkdocs
+ mkdocs-autorefs
mkdocs-material
```